### ISSUE TITLE
executing the playthrough task throws error: AttributeError: attempted to get missing private attribute '_max_episode_steps'

### ISSUE THREAD

**Issue by bartman081523** [MineDojo/MineDojo#19]
sample_script.py:
```
import minedojo

env = minedojo.make(
    task_id="playthrough",
    image_size=(640, 1024)
)
obs = env.reset()
for i in range(10000):
    act = env.action_space.no_op()
    obs, reward, done, info = env.step(act)
env.close()
```

```
JAVA_HOME=/usr/lib/jvm/java-8-openjdk/ vglrun python3 sample_script.py
[INFO:minedojo.tasks] Loaded 1572 Programmatic tasks, 1558 Creative tasks, and 1 special task: "Playthrough". Totally 3131 tasks loaded.
[INFO:minedojo.tasks] Loaded 1572 Programmatic tasks, 1558 Creative tasks, and 1 special task: "Playthrough". Totally 3131 tasks loaded.
Traceback (most recent call last):
  File "/home/user/Downloads/minedojo/MineDojo/scripts/2sample_script.py", line 13, in <module>
    obs, reward, done, info = env.step(act)
  File "/home/user/miniconda3/envs/minedojo/lib/python3.9/site-packages/gym/core.py", line 289, in step
    return self.env.step(action)
  File "/home/user/miniconda3/envs/minedojo/lib/python3.9/site-packages/minedojo/sim/wrappers/ar_nn/delta_inventory_wrapper.py", line 85, in step
    observation, reward, done, info = self.env.step(action)
  File "/home/user/miniconda3/envs/minedojo/lib/python3.9/site-packages/gym/core.py", line 323, in step
    observation, reward, done, info = self.env.step(action)
  File "/home/user/miniconda3/envs/minedojo/lib/python3.9/site-packages/minedojo/sim/wrappers/ar_nn/nn_action_space_wrapper.py", line 274, in step
    obs, reward, done, info = self.env.step(malmo_action)
  File "/home/user/miniconda3/envs/minedojo/lib/python3.9/site-packages/minedojo/tasks/meta/playthrough.py", line 179, in step
    if self._elapsed_steps >= self._max_episode_steps:
  File "/home/user/miniconda3/envs/minedojo/lib/python3.9/site-packages/gym/core.py", line 235, in __getattr__
    raise AttributeError(
AttributeError: attempted to get missing private attribute '_max_episode_steps'
```

Setup & validation:
```
sudo pacman -S  jdk8-openjdk
conda create -y -n minedojo python=3.9
conda activate minedojo
git clone https://github.com/MineDojo/MineDojo
cd MineDojo
pip install -e .

JAVA_HOME=/usr/lib/jvm/java-8-openjdk/ vglrun python3 scripts/validate_install.py
[INFO:minedojo.tasks] Loaded 1572 Programmatic tasks, 1558 Creative tasks, and 1 special task: "Playthrough". Totally 3131 tasks loaded.
/home/user/miniconda3/envs/minedojo/lib/python3.9/site-packages/gym/spaces/box.py:73: UserWarning: WARN: Box bound precision lowered by casting to float32
  logger.warn(
[INFO] Create a task with prompt: combat a spider in night plains with a diamond sword, shield, and a full suite of leather armors
[INFO:minedojo.tasks] Loaded 1572 Programmatic tasks, 1558 Creative tasks, and 1 special task: "Playthrough". Totally 3131 tasks loaded.
[INFO] Installation Success
```

```
cat /etc/os-release
NAME="Manjaro Linux"
ID=manjaro
ID_LIKE=arch
BUILD_ID=rolling
PRETTY_NAME="Manjaro Linux"
ANSI_COLOR="32;1;24;144;200"
HOME_URL="https://manjaro.org/"
DOCUMENTATION_URL="https://wiki.manjaro.org/"
SUPPORT_URL="https://manjaro.org/"
BUG_REPORT_URL="https://bugs.manjaro.org/"
LOGO=manjarolinux
```

**Comment by yunfanjiang** [comment]
Hey @bartman081523 , thank you for reporting this. It is indeed a bug. We've fixed it per 8adaa4a51c2d499ef4e77c1f7e62339eca685638

**Comment by bartman081523** [comment]
Thanks, that was blazingly fast :-D
I also think that I am confusing MineDojo with MineRL with VPT pretrained model. 
Therefore it must be wanted behaviour, that this sample_script does nothing, besides initializing the minecraft server and standing on the spawn (`act = env.action_space.no_op()`)
